guides/v2.2/cloud/project/project-conf-files_services-mysql.md

```diff
@@ -108,3 +94,61 @@ relationships:
 
 {: .bs-callout-info }
 If you configure one MySQL user, you cannot use the [`DEFINER`](http://dev.mysql.com/doc/refman/5.6/en/show-grants.html) access control mechanism for stored procedures and views.
+
+## Connect to the database
+
+Accessing the MariaDB database directly requires you to use a SSH to log in to the remote server, and connect to the database.
+
+
+1. Log in to the remote server using SSH.
+
+1. Retrieve the MySQL login credentials from the `database` and `type` properties in the [$MAGENTO_CLOUD_RELATIONSHIPS]({{ page.baseurl }}/cloud/project/project-conf-files_magento-app.html#relationships) variable:
+
+   ```bash
+   echo $MAGENTO_CLOUD_RELATIONSHIPS | base64 -d | json_pp
```

Comment:
there are 2 more ways to retrieve this information and I would say that their more convinient:
 - after connecting through SSH - `vendor/bin/ece-tools env:config:show services`
 - on the local env - `mgc relationships`

Reply:
Created MAGECLOUD-4515 to improve the documentation for retrieving service relationship information.